java/src/com/ibm/streamsx/topology/kafka/KafkaMessageImpl.java

```diff
@@ -0,0 +1,119 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2015  
+ */
+package com.ibm.streamsx.topology.kafka;
+
+import java.io.Serializable;
+
+/**
+ * Stream object class usable with KafkaStreams sources.
+ * 
+ * @see KafkaStreams
+ */
+public class KafkaMessageImpl implements Serializable, KafkaMessage {
```

Comment:
Is there any benefit to having a impl/interface split, it seems that the expectation is that developers would use KafkaMessageImpl.

If a split is needed, then maybe a better name, I tend to assume Impl classes are not part of the public api.